### PR TITLE
Ticket6777

### PR DESCRIPTION
**Pull request by boeuf-codes**
### Description of work

Created OPI for automatic needle valve controller. 

### Ticket

[Ticket #6777](https://github.com/ISISComputingGroup/IBEX/issues/6777)

### Acceptance criteria

See ticket.

### Documentation

See documentation [here](https://github.com/ISISComputingGroup/ibex_developers_manual/wiki/Automatic-Needle-Valve-Controller) for some behaviours of the OPI 

---

#### Code Review

- [ ] Is the code of an acceptable quality?
- [ ] If the change is to an OPI, does the `check_opi_format.py` script in [C:\Instrument\Dev\ibex_gui\base\uk.ac.stfc.isis.ibex.opis](https://github.com/ISISComputingGroup/ibex_gui/blob/master/base/uk.ac.stfc.isis.ibex.opis/check_opi_format.py) pass?
- [ ] Do the changes function as described and is it robust?
- [ ] Is there associated PR for the [release notes](https://github.com/ISISComputingGroup/IBEX/blob/master/release_notes/ReleaseNotes_Upcoming.md)?

### Final Steps
- [ ] Reviewer has also merged the [release notes](https://github.com/ISISComputingGroup/IBEX/blob/master/release_notes/ReleaseNotes_Upcoming.md)

### CODE REVIEW

File: base/uk.ac.stfc.isis.ibex.opis/resources/opi_info.xml [hunk 1]
```diff
@@ -2916,6 +2916,27 @@
         </categories>
       </value>
     </entry>
+    <entry>
+      <key>Automatic Needle Valve Controller</key>
+      <value>
+        <type>UNKNOWN</type>
```
Comment (Tom-Willemsen) [inline base/uk.ac.stfc.isis.ibex.opis/resources/opi_info.xml:2922]: I think temperature controller or eurotherm would be better than `UNKNOWN` here.

File: base/uk.ac.stfc.isis.ibex.targetplatform/targetplatform.target [hunk 2]
```diff
@@ -227,7 +221,7 @@
 			</dependency>
 		</dependencies>
 	</location>
-	<location includeDependencyDepth="none" includeSource="true" missingManifest="generate" type="Maven">
+	<location includeDependencyDepth="none" includeDependencyScopes="compile" includeSource="true" missingManifest="generate" type="Maven">
```
Comment (Tom-Willemsen) [inline base/uk.ac.stfc.isis.ibex.targetplatform/targetplatform.target:224]: Was this change intentional or not?

**Review by zsoltkebel — APPROVED** [review]
